test/test_create_db.py

```diff
@@ -0,0 +1,59 @@
+###############################################################
```

Comment:
Ultimately, I understand these test cases will be hooked into a high level test driver like `make check` equivalent. But for the time being, it would be good to have a short README.md file explaining how to run this properly? Don't have to be fancy though. 

You can add a couple of sentences in your top-level the README.md file as well. The first thing a developer would like to do would typically be to run a "hello world" type test and documenting such in README.md goes a long way towards helping them ensure things are working after git clone, make, make install or equivalents.